src/helm/benchmark/adaptation/adapter_spec.py

```diff
@@ -71,6 +73,9 @@ class AdapterSpec:
     # set of training instances.  Used to compute error bars.
     num_train_trials: int = 1
 
+    # Number of trials, where we query the model with the exact same requests
```

Comment:
Clarify that a different seed is used for each request - so it is not the "exact same" request.